src/schema/v2/sale/__tests__/index.test.ts

```diff
@@ -847,6 +847,7 @@ describe("Sale type", () => {
         start_at: "2022-03-09 09+07:00",
         end_at: "2022-03-12 09+07:00",
         cascading_end_time_interval: 120,
+        cascading_end_time_interval_minutes: 2,
```

Comment:
Spec that requests the new field (for a sale w/ and w/o cascading) might be nice!